ios/brave-ios/Sources/Brave/Frontend/Sync/SyncAddDevice/SyncAddDeviceDetailView.swift

```diff
@@ -0,0 +1,483 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveCore
+import BraveShared
+import Shared
+import UIKit
+
+class SyncAddDeviceInformationView: UIStackView {
+
+  private var titleLabel = UILabel().then {
+    $0.translatesAutoresizingMaskIntoConstraints = false
+    $0.font = UIFont.systemFont(ofSize: 22, weight: UIFont.Weight.semibold)
```

Comment:
Can it be `$0.font = .preferredFont(forTextStyle: ....)`?

Reply:
Fix at https://github.com/brave/brave-core/pull/23394/commits/51acf862c997441047ace17712c0fc80e726ca75